### PR TITLE
[Addition_layer] Set output to zero in forward

### PR DESCRIPTION
**Pull request by lhs8928**
Before adding inputs output should be set zero

Self evaluation:

Build test: [X]Passed [ ]Failed [ ]Skipped
Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: hyeonseok lee <hs89.lee@samsung.com>

### CODE REVIEW

File: nntrainer/layers/addition_layer.cpp [hunk 1]
```diff
@@ -45,6 +45,8 @@ void AdditionLayer::forwarding(bool training) {
   Tensor &hidden_ = net_hidden[0]->getVariableRef();
   TensorDim &in_dim = input_dim[0];
 
+  hidden_.setZero();
```
Comment (kparichay) [inline nntrainer/layers/addition_layer.cpp:48]: good catch.
consider using `hidden_.add(net_input[0]->getVariableRef())` for the first iteration, and `hidden_.add_i(net_input[idx]->getVariableRef())` for the remaining inputs.

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: nntrainer/layers/addition_layer.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/layers/addition_layer.cpp:48]: Could you check https://github.com/nnstreamer/nntrainer/pull/1096#discussion_r612147251 and fix accordingly?

Comment (lhs8928) [reply nntrainer/layers/addition_layer.cpp]: Done

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM :)

**Pull request merged**